src/main/java/seedu/cafectrl/parser/Parser.java

```diff
@@ -70,7 +70,7 @@ public class Parser implements ParserUtil {
     /** The rest of Command Handler Patterns*/
     private static final String LIST_INGREDIENTS_ARGUMENT_STRING = "(\\d+)";
     private static final String DELETE_ARGUMENT_STRING = "(\\d+)";
-    private static final String EDIT_PRICE_ARGUMENT_STRING = "index/(\\d+) price/(\\d+(\\.\\d+)?)";
+    private static final String EDIT_PRICE_ARGUMENT_STRING = "index/(.*) price/(.*)";
```

Comment:
I noticed that this regex still accepts more than 2 d.p. numbers,
![image](https://github.com/AY2324S1-CS2113-T17-2/tp/assets/35828587/a000166b-1ab2-43f3-be00-d455dd4a1e47)

I remember you said you plan to round it up to 2 d.p. , can I check if this was implemented in this PR?  if not, another suggestion I can give is to use this regex (inspired from my add dish regex) which can reject inputs more than 2 d.p. but can allow the user to enter prices up to 2 d.p. (e.g. price/2 OR price/2.0 OR price/2.01 OR price/.01)

`"index/([0-9]*) price/([0-9]*\\.[0-9]{0,2}|[0-9]+)$"`

Reply:
If you do use my regex remember to do extensive testing :P

Reply:
I implemented the round up feature in dish feature, so no matter how many dps are provided by the user the app will automatically update to 2dp